sdk/face/ai-vision-face-rest/test/public/faceSessionClient.spec.ts

```diff
@@ -0,0 +1,562 @@
+      .get();
+    if (isUnexpected(getLivenessSessionsResponse)) {
+      throw new Error(getLivenessSessionsResponse.body.error.message);
+    }
+    assert.equal(getLivenessSessionsResponse.status, "200");
+    console.log(`Number of sessions: ${getLivenessSessionsResponse.body.length}`);
+    const deleteLivenessSessionResponses = await Promise.all(
+      getLivenessSessionsResponse.body.map((livenessSessionItemOutput) =>
+        client
+          .path(
+            "/detectLivenessWithVerify/singleModal/sessions/{sessionId}",
+            livenessSessionItemOutput.id,
+          )
+          .delete(),
+      ),
+    );
+    assert.isTrue(deleteLivenessSessionResponses.every((response) => response.status === "200"));
+  });
+
+  it("TestCreateVerifySession", async () => {
+    const deviceCorrelationId = recorder.variable("deviceCorrelationId", randomUUID());
+    const createLivenessSessionResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions")
+      .post({
+        body: {
+          livenessOperationMode: "Passive",
+          deviceCorrelationId,
+        },
+      });
+    if (isUnexpected(createLivenessSessionResponse)) {
+      throw new Error(createLivenessSessionResponse.body.error.message);
+    }
+    assert.equal(createLivenessSessionResponse.status, "200");
+    assert.isNotEmpty(createLivenessSessionResponse.body.sessionId);
+    assert.isNotEmpty(createLivenessSessionResponse.body.authToken);
+
+    const { sessionId } = createLivenessSessionResponse.body;
+
+    const getLivenessSessionResultResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions/{sessionId}", sessionId)
+      .get();
+    if (isUnexpected(getLivenessSessionResultResponse)) {
+      throw new Error(getLivenessSessionResultResponse.body.error.message);
+    }
+    assert.equal(getLivenessSessionResultResponse.status, "200");
+    assert.equal(getLivenessSessionResultResponse.body.deviceCorrelationId, deviceCorrelationId);
+
+    const deleteLivenessSessionResponse = await client
+      .path("/detectLivenessWithVerify/singleModal/sessions/{sessionId}", sessionId)
+      .delete();
+    if (isUnexpected(deleteLivenessSessionResponse)) {
+      throw new Error(deleteLivenessSessionResponse.body.error.message);
+    }
+    assert.equal(deleteLivenessSessionResponse.status, "200");
+  });
+
+  // TODO: readFileSync is not available in browser environment.
+  // it.runIf(isLiveMode())("TestCreateVerifySessionWithImage", async () => {
+  //   const deviceCorrelationId = recorder.variable("deviceCorrelationId", randomUUID());
+  //   const createLivenessSessionResponse = await client
+  //     .path("/detectLivenessWithVerify/singleModal/sessions")
+  //     .post({
+  //       contentType: "multipart/form-data",
+  //       body: [
+  //         {
+  //           name: "VerifyImage",
+  //           body: readFileSync("samples-dev/data/detection1.jpg"),
```

Comment:
@timovv Any suggestion on how to write mfd test case which could compile and run in both environments?

Reply:
No, `createFile` is not used anymore. There isn't an easy way since it is difficult to reference a file that exists on disk in the browser. The only consistent way I can think of is to embed the file in the test (base64 encode it and store it as a constant, then decode it and convert it to a `Uint8Array` somehow, which is supported in both environments). That's OK for small files but for large files like an image it isn't great.

My recommendation for now would be to make this test Node-only (putting it in a separate file in `test/public/node` should do it). Then we can add comments on how you would use it in browser. We don't really need to test both cases thoroughly here I think; we have coverage already in Core for that.

Reply:
@chungshengfu Could you move this case this folder?

I would expect in browser we would skip the build for that case: https://github.com/chungshengfu/azure-sdk-for-js/blob/user/chfu/ai-vision-face/sdk/face/ai-vision-face-rest/tsconfig.browser.config.json#L4.